electricitylci/petroleum_upstream.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+import pandas as pd
+from electricitylci.coal_upstream import read_eia923_fuel_receipts
+from os.path import join
+from electricitylci.globals import data_dir, output_dir
+import electricitylci.PhysicalQuantities as pq
+
+def generate_petroleum_upstream(year):
+    """
+    Generate the annual petroleum extraction, transport, and refining emissions
+    (in kg) for each plant in EIA923.
+    
+    Parameters
+    ----------
+    None
```

Comment:
Add year parameter. It's pretty obvious what it's supposed to be but better to not have the conflicting info.

```python
year : int
    Year of EIA-923 fuel data to use
```